### ISSUE TITLE
An exception has occurred, use %tb to see the full traceback. and low accuracy

### ISSUE THREAD

**Issue by vilmaunjan** [tensorflow/tensorflow#13432]
![tf issue](https://user-images.githubusercontent.com/22562558/31056287-282fb44e-a69d-11e7-800b-af2a8ab41f12.png)

The accuracy I get for sklearn and tensorflow is 0.428571. And after that I get a line that says: An exception has occurred, use %tb to see the full traceback.

What should I do? What is the accuracy that I am supposed to get?

**Comment by asimshankar** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. Thank you.
